lint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+printf "[FLAKE8: project-liberation]\n"
+flake8                                                                                          \
+    --exclude=project_liberation/project_liberation/settings/,project_liberation/manage.py,                                           \
+    --jobs=4                                                                                    \
+    --max-line-length=120                                                                       \
+    --ignore=E124,E126,E128,E131,E156,E201,E221,E222,E241,E265,E271,E272,E701,F405,E501,W503    \
+
+printf "\n"
+
+printf "[PYLINT: project-liberation]\n"
+# Find all subdirectories of our python apps and use xargs to pass them as arguments to pylint
+
+./find-files-to-refactor.sh | xargs pylint --rcfile=pylintrc
```

Comment:
What do you mean by "files to refactor"? Are you really going to be refactoring them? Why not run pylint on all relevant files?

Reply:
This name better refer to what black and isort do with this files. I change it to more proper name.

Reply:
Name changed. I think this name is more relevant